.github/dependabot.yml

```diff
@@ -22,10 +24,10 @@ updates:
       - "dependency"
       - "gradle"
     commit-message:
-      prefix: "[No Ticket]"
+      prefix: "[WOR-1448]"
     ignore:
       - dependency-name: "org.springframework.boot:spring-boot-gradle-plugin"
-        update-types: ["version-update:semver-major"]
+        update-types: ["version-update:semver-major", "version-update:semver-minor"]
       - dependency-name: "org.springframework.boot:spring-boot-starter-data-jdbc"
         update-types: ["version-update:semver-patch", "version-update:semver-minor"]
```

Comment:
The existing ignore rules are set to ignore minor and **patch** updates, is that intentional?

Reply:
Yes. We are hitting problems with spring-boot both with a minor version upgrade and a major one.

Reply:
@cahrens then shouldn't we be ignoring major versions instead of ignoring patch versions?

Reply:
Sorry, the issues I saw were with both minor and patch upgrades… I got the semantic stuff messed up. That was for `spring-boot-gradle-plugin`.

But now I see your comment is actually `spring-boot-starter-data-jdbc`. I think that all of these should actually be ignoring both patch and minor, and with my most recent [PR](spring-boot-starter-data-jdbc) they will all be. Sorry for the churn on this.